specs/agents/log-sending.md

```diff
@@ -0,0 +1,61 @@
+# Log sending
+
+### `log_sending` configuration
+
+**Warning**: experimental feature, may be subject to change until GA. Also, only a small subset of agents will provide it before GA.
+
+Controls the ability to send logs directly from the agent to APM server.
+
+|                |                 |
+|----------------|-----------------|
+| Valid options  | `true`, `false` |
+| Default        | `false`         |
+| Dynamic        | `true`          |
+| Central config | `true`          |
+
+When set to `true`, the agent will duplicate log events and send them to apm-server.
+Original log events are unaltered and written to their usual destinations (file, stdout, ...).
+
+The APM server only supports log events as of version 8.6+, thus trying to use this with an older version should
+issue a warning/error in the agent logs.
+
+### Log event format
+
+On the agent side, there are two ways to get an ECS-formatted log message from a log event:
+- The application already uses [ecs-logging](https://github.com/elastic/ecs-logging)
+- The agent embeds a copy of [ecs-logging](https://github.com/elastic/ecs-logging), which might also be used for [log reformatting](./log-reformatting.md).
+
+In both cases, the output of [ecs-logging](https://github.com/elastic/ecs-logging) can be reused as follows:
+
+```
+{"log":<ecs-formatted-log-event>}\n`
+```
```

Comment:
Should we be expecting a "log.json" at https://github.com/elastic/apm-server/tree/main/docs/spec/v2/ ?  I guess there isn't a particular *need* for it.

Reply:
We used to have a `log.json` schema but removed it. See also https://github.com/elastic/apm-server/pull/9349#discussion_r996667527

One of the reasons is that basically any top-level properties should be allowed eventually and that dotted and nested notations are treated the same which can't easily be expressed in a JSON schema definition.